src/views/virtualmachines/list/hooks/useExistingSelectedVMs.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getName, getNamespace } from '@kubevirt-utils/resources/shared';
+
+import { VMIdentifier } from '../selectedVMs';
+
+const useExistingSelectedVMs = (vms: V1VirtualMachine[], selectedVMs: VMIdentifier[]) => {
```

Comment:
`selectedVMs` is a signal and can be directly imported in this file without having it as a input.
We can use this hook in the `VirtualMachineBulkActionButton` component and also in the `VirtualMachinesList`